lib/geocoder/lookups/location_iq.rb

```diff
@@ -8,7 +8,7 @@ def name
     end
 
     def use_ssl?
-      true
+      ["use_ssl"]
```

Comment:
Sorry, I misspoke here. We actually shouldn't be implementing `use_ssl?` in lookup subclasses. `supported_protocols` should be used instead. But since the default is HTTP and HTTPS, this method can simply be deleted.

Reply:
Removed `use_ssl?`